tests/test_coin_combination.py

```diff
@@ -1,3 +1,17 @@
 from app.coin_combination import get_coin_combination
 
-# write your tests here
+
+def test_zero_cents():
+    assert get_coin_combination(0) == [0, 0, 0, 0]
+
+
+def test_all_types_coins():
+    assert get_coin_combination(42) == [2, 1, 1, 1]
+
+
+def test_one_coin():
```

Comment:
Could you also write tests when the answer contains coins only of one specific type (nickels, dimes, quarter)?